packaging/deb/rules

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/make -f
+# You must remove unused comment lines for the released package.
+#export DH_VERBOSE = 1
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+#export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+#export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
```

Comment:
These should be removed, but add a new wiki article that describes how to debug the debian build process. `DH_VERBOSE=1` is particularly helpful.